pkgs/applications/misc/iterm2/default.nix

```diff
@@ -1,31 +1,48 @@
 { stdenv, fetchFromGitHub }:
 
+ /*
+ This derivation is impure: it relies on an Xcode toolchain being installed
+ and available in the expected place. The values of sandboxProfile
+ are copied pretty directly from the MacVim derivation, which
+ is also impure. In order to build you at least need the `build-use-chroot`
+ option set to `relaxed` or set the `sandbox` option to `false`.
```

Comment:
What is `build-use-chroot` and where is it documented?

Reply:
This is the old name for the `sandbox` option.

Reply:
@earksiinni Sorry about the confusion, I corrected my comment in a second commit.